.monorepo.json

```diff
@@ -45,6 +45,22 @@
       "version": "0.3.1",
       "hash": "d99e730f5b1c9e8b585fb44ee61987c6"
     },
+    "@angular-devkit/architect": {
```

Comment:
Please name `architect-cli` and name the package `architect`.

Reply:
Fixed